### DISCUSSION TITLE
realese 下载的 3.95.81-v1 版本无法注入，换了好几种注入方法dll都没有注入成功

### DISCUSSION THREAD

**Discussion by vjeson** [ttttupup/wxhelper#203] [Q&A]
realese 下载的 3.95.81-v1 版本无法注入，换了好几种注入方法dll都没有注入成功

请问release 版本最新版本是x64位的么？

自己编译一直报错，好像是spdlog 链接不对，不知道怎么解决

unresolved external symbol "public: __cdecl spdlog

**Comment by ttttupup** [comment]
64位的

**Comment by vjeson** [comment]
你好，编译教程都是编译x86的，按照教程修改编译64一直在好像链接spdlog的时候报错？能否给一点指导，3.9.5.81一直无法注入成功，之前的，都是可以的

**Comment by ttttupup** [comment]
3.9.5.81分支有编译64位的说明。spdlog需要先clone下来。

**Comment by caiyakang** [comment]
一注入就闪退,请问有什么办法可以解决呢.

**Reply by muxue2333** [reply]
解决了吗，我也注入就闪退

**Comment by ttttupup** [comment]
#65
